politeiawww/backend.go

```diff
@@ -1621,21 +1608,19 @@ func (b *backend) ProcessProposalDetails(propDetails www.ProposalsDetails, user
 			Signature:        cachedProposal.Signature,
 			CensorshipRecord: cachedProposal.CensorshipRecord,
 			NumComments:      cachedProposal.NumComments,
-			UserId:           cachedProposal.UserId,
-			Username:         b.getUsernameById(cachedProposal.UserId),
```

Comment:
Why is username removed from the proposal details?

Reply:
Totally my mistake. I didn't realize I broke it with the rebase.